documentation/src/main/asciidoc/topics/upgrading.adoc

```diff
@@ -27,6 +27,7 @@ The following defaults have changed for the Hot Rod client:
 * `socket_timeout` is now 2000 ms / 2 seconds (previously 60000 ms / 60 seconds)
 * `max_retries` is now 3 (was 10)
 * `min_evictable_idle_time` is now 180000 ms / 3 minutes (previously 1800000 ms / 30 minutes)
+* `ssl_hostname_validation` is a new property and defaults to `true`. This also requires setting the `sni_hostname`.
```

Comment:
Should we also state that the server-side certs may also have to be updated?

Reply:
Yes, I'm also going to add a warning on server startup if the certificate doesn't have the `SubjectAlternativeNamesExtension`